docroot/legalcode/by-nc-nd_4.0.html

```diff
@@ -312,7 +312,7 @@ <h3>Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International P
+              <p class="shaded"><a id="languages"></a>Additional languages available: <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.id">Bahasa Indonesia</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.es">Español</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.eu">euskara</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.de">Deutsch</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.es">Español</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.fr">français</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.hr">hrvatski</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.it">italiano</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.lv">latviski</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.lt">Lietuvių</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.nl">Nederlands</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.no">norsk</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.pl">polski</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.pt">português</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.fi">suomeksi</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.sv">svenska</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.mi">te reo Māori</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.tr">Türkçe</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.cs">čeština</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.el">Ελληνικά</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.ru">русский</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.uk">українська</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.ar">العربية</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.zh-Hans">中文</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.ja">日本語</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.zh-Hant">華語</a>, <a href="//creativecommons.org/licenses/by-nc-nd/4.0/legalcode.ko">한국어</a>. Please read the <a href="//wiki.creativecommons.org/FAQ#officialtranslations">FAQ</a> for more information about official translations.</p>
```

Comment:
It's a bit strange to have an empty `<a>` tag just to link from the top of the page. It might not be very accessible (not entirely sure how screen readers will behave with that.

Isn't it better to have something like this?
`<h4 id="languages">Additional languages available:</h4>`

Reply:
Fair enough :)